lib/resources/file.rb

```diff
@@ -44,7 +44,7 @@ def initialize(path)
     %w{
       type exist? file? block_device? character_device? socket? directory?
       symlink? pipe? mode mode? owner owned_by? group grouped_into?
-      link_path linked_to? mtime size selinux_label immutable?
+      link_path linked_to? mtime size size_lines selinux_label immutable?
```

Comment:
There is no need to add this here, since the implementation is done in this file anyway